queue-rabbitmq/src/main/java/com/cisco/oss/foundation/message/RabbitMQMessagingFactory.java

```diff
@@ -98,7 +98,7 @@ public void run() {
                             if (message.ack)
                                 channel.basicAck(message.deliveryTag, false);
                             else
-                                channel.basicNack(message.deliveryTag, false, true);
+                                channel.basicNack(message.deliveryTag, false, false);
```

Comment:
Yes configuration I will add

From: Yair Ogen [mailto:notifications@github.com]
Sent: Monday, January 25, 2016 11:43 AM
To: foundation-runtime/communication <communication@noreply.github.com>
Cc: Izek Greenfield (igreenfi) <igreenfi@cisco.com>
Subject: Re: [communication] support changing DLX queue name (#50)


In queue-rabbitmq/src/main/java/com/cisco/oss/foundation/message/RabbitMQMessagingFactory.java<https://github.com/foundation-runtime/communication/pull/50#discussion_r50671640>:

> @@ -98,7 +98,7 @@ public void run() {

>                              if (message.ack)

>                                  channel.basicAck(message.deliveryTag, false);

>                              else

> -                                channel.basicNack(message.deliveryTag, false, true);

> +                                channel.basicNack(message.deliveryTag, false, false);

why change the requeue from true to false? should it be configurable?

—
Reply to this email directly or view it on GitHub<https://github.com/foundation-runtime/communication/pull/50/files#r50671640>.